ouroboros-consensus/src/unstable-mock-block/Ouroboros/Consensus/Mock/Ledger/Address.hs

```diff
@@ -28,6 +30,12 @@ newtype Addr = Addr String
     , NoThunks
     )
 
+instance ToCBOR Addr where
+  toCBOR (Addr a) = toCBOR $ pack a
+
+instance FromCBOR Addr where
+  fromCBOR = Addr . unpack <$> fromCBOR
```

Comment:
No need to (un)pack as `{From,To}CBOR Text` exists.

Reply:
But I'm doing it the other way round. Addr is `Addr String`, and I need to convert that to CBOR, so I'm doing it through `Text`

Reply:
Ah, I can't read :facepalm: